setup.py

```diff
@@ -29,5 +29,8 @@
             "flash-attn",
             "deepspeed",
         ],
+        "peft": [
+            "peft @ git+https://github.com/huggingface/peft.git",
+        ],
```

Comment:
we do it this way b/c we don't install Peft with setup.py as the gptq variant has a different Peft version pinned.

Reply:
Should we just add the pip install line to the `test.yml` instead of adding it here?

Reply:
Time to do another poll on github maybe?